app-misc/tdl/tdl-1.5.2-r4.ebuild

```
@@ -0,0 +1,80 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit edo toolchain-funcs
+
+MY_PV="$(ver_rs 1- '_')"
+DESCRIPTION="Command line To Do List manager"
+HOMEPAGE="https://github.com/rc0/tdl"
+SRC_URI="https://github.com/rc0/tdl/archive/V${MY_PV}/V${MY_PV}.tar.gz -> ${PF}.tar.gz"
```

I suggest avoiding `${PF}` in `SRC_URI`, because it will backfire when the next revbump is done. How about the following instead?

```suggestion
SRC_URI="https://github.com/rc0/tdl/archive/V${MY_PV}/V${MY_PV}.tar.gz -> ${P}.gh.tar.gz"
```

---

Sounds good, just updated the PR. :)